src/main/clientHandler/ClientHandlerThread.java

```diff
@@ -19,8 +24,20 @@ public void run() {
         } catch (UnexpectedActionException e) {
             e.printStackTrace();
         }
-        // Currently, we always get messages with id 1
+        // TODO Currently, we always get messages with id 1, change this to support multiple ids
         assert(actionToPerform != null && actionToPerform.getTargetId() == 1);
```

Comment:
According to [Oracle](http://docs.oracle.com/javase/7/docs/technotes/guides/language/assert.html):

> Do not use assertions to do any work that your application requires for correct operation.

This because assertions are not enabled by default in the JVM